### ISSUE TITLE
关于同一 batch cross modal输入的问题

### ISSUE THREAD

**Issue by Harzva** [linzhiqiu/cross_modal_adaptation#3]
if image_feature is not None and text_feature is not None:
            feature = torch.cat([image_feature, text_feature], dim=0)
            label = torch.cat([image_label, text_label], dim=0)
        elif image_feature is not None:
            feature = image_feature
            label = image_label
        elif text_feature is not None:
            feature = text_feature
            label = text_label
        else:
            raise ValueError("Both image_feature and text_feature are None")
实际上我们的batch大小等于config里的2倍，这样理解对吗

**Comment by linzhiqiu** [comment]
Yes that is correct.

**Comment by Harzva** [comment]
感谢大佬的回答。还有一个小问题
```
    elif image_feature is not None:
        feature = image_feature
        label = image_label
    elif text_feature is not None:
        feature = text_feature
        label = text_label
```
        这个elif是为了做消融实验吗，实际上训练的时候都是以下代码可以这样理解吗
```
       if image_feature is not None and text_feature is not None:
        feature = torch.cat([image_feature, text_feature], dim=0)
        label = torch.cat([image_label, text_label], dim=0)
```

**Comment by linzhiqiu** [comment]
Yes that is correct.
